### ISSUE TITLE
Text elements must have sufficient color contrast against the background        

### ISSUE THREAD

**Issue by bmbc-dev-detec** [LocalGovIMS/localgov-ims#325]
WCAG Reference(s):
1.4.3 Contrast (Minimum)
- The blue links at the top of the page have insufficient colour contrast.
- Placeholder text in form fields have insufficient colour contrast.
- Buttons with a bright green or blue background and white text have insufficient colour contrast.
- Days and numbers within the datepicker popup have insufficient colour contrast.
- The component class class="ui info message" uses blue text with insufficient colour contrast against the blue background.
- The accordion component has insufficient text colour contrast on the toggles.
- In the Edit eReturn page there's insufficient text colour contrast on the white text with a teal background
- In the Make a payment page, the Empty basket link has insufficient text colour contrast

**Comment by bmbc-dev-detec** [comment]
Ready for retesting after completion of #416

**Comment by bmbc-dev-detec** [comment]
Closing. Will add any problem found after retesting as new issues
